server/routes/message.route.js

```diff
@@ -8,11 +8,21 @@ const router = express.Router(); // eslint-disable-line new-cap
 router.route('/send')
     .post(validate(paramValidation.sendMessage), messageCtrl.send);
 
-// TODO: needs to be clear that userId is the owner
-router.route('/list/:userId')
+/**
+ * url params:
+ * - limit: number of messages to return
+ * - from: filter on message sender by username
+ * - summary: boolean, can return a summary version of messages
+ */
+router.route('/list/')
     .get(messageCtrl.list);
-// TODO: needs to be clear that userId is the owner
-router.route('/count/:userId')
+
+/**
+ * url params:
+ * - owner: username of owner of messages to count
```

Comment:
@RuchitaGarde why does `count` need an owner but `list` does not? Shouldn't the username/userId be coming from the JWT in both cases?

Additionally, do you want to modify this code now to user `userId` instead of `username`, or did you want to do that in a separate PR?

Reply:
1) `count` does not need an owner. That is my mistake. The `owner` param needs to be removed.
2) I would like to do that in a separate PR. I don't want to mix things up right now. I will do that after this branch gets merged with develop.